schema/tables/account_policy_data.yml

```
@@ -1,5 +1,13 @@
 name: account_policy_data
-description: Additional macOS user account data from the AccountPolicy section of [OpenDirectory](https://en.wikipedia.org/wiki/Apple_Open_Directory).
+description: Additional macOS user account data from the AccountPolicy section of [OpenDirectory](https://en.wikipedia.org/wiki/Apple_Open_Directory), the identity provider used by Apple.
+columns:
+  - name: uid
+    description: [User ID](https://superuser.com/a/1108201)
```

```suggestion
    description: "[User ID](https://superuser.com/a/1108201)"
```

@mikermcneil it looks like the YAML parser may be interpreting this value as an array. You can wrap the value in quotes to prevent this.
cc: @nonpunctual

---

Can you push it through so we can see if the link works?